common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -1195,31 +1201,84 @@ def _block_matches_all(block_data):
         # don't expect caller to know that children are in fields
         if 'children' in qualifiers:
             settings['children'] = qualifiers.pop('children')
+
+        # No need of these caches unless include_orphans is set to False
+        path_cache, parents_cache = None, None
```

Comment:
we won't actually even use these variables if `include_orphans` is set to `False`. Maybe not even worth setting?